src/bmm-skills/4-implementation/bmad-agent-dev/scripts/resolve-customization.py

```
@@ -0,0 +1,182 @@
+            return None
+        current = parent
+
+
+def load_toml(path: Path) -> dict[str, Any]:
+    """Return parsed TOML or empty dict if the file doesn't exist."""
+    if not path.is_file():
+        return {}
+    try:
+        with open(path, "rb") as f:
+            return tomllib.load(f)
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)